src/test/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+# ===============================
+# DATABASE
```

Comment:
are we doing integration tests?

Reply:
For this Rest Repo, only integration test.

Reply:
👍